code/modules/client/preferences_savefile.dm

```diff
@@ -112,6 +112,22 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	if(current_version < 24)
 		if (!(underwear in GLOB.underwear_list))
 			underwear = "Nude"
+	if(current_version < 25)
+		if(!(randomise.len))
+			randomise += list(RANDOM_NAME = FALSE, RANDOM_BODY = FALSE, RANDOM_SPECIES = FALSE, RANDOM_GENDER = FALSE, RANDOM_AGE = FALSE, RANDOM_UNDERWEAR = TRUE, RANDOM_UNDERWEAR_COLOR = TRUE, RANDOM_UNDERSHIRT = TRUE, RANDOM_SOCKS = TRUE, RANDOM_BACKPACK = TRUE, RANDOM_JUMPSUIT_STYLE = TRUE, RANDOM_HAIRSTYLE = TRUE, RANDOM_HAIR_COLOR = TRUE, RANDOM_FACIAL_HAIRSTYLE = TRUE, RANDOM_FACIAL_HAIR_COLOR = TRUE, RANDOM_SKIN_TONE = TRUE, RANDOM_EYE_COLOR = TRUE)
```

Comment:
@anturk so this didn't work
the randomisation works fine as a guest (so no savefile) but adding this will not populate the randomise list of clients with existing savefiles and so none of the buttons that access it work

Reply:
pls help